.storybook/main.js

```diff
@@ -10,9 +12,35 @@ module.exports = {
     );
     svelteLoader.options = {
       ...svelteLoader.options,
-      preprocess: sveltePreprocess({ postcss: true }),
+      preprocess: sveltePreprocess({
+        postcss: true,
+        defaults: {
+          style: "scss",
+        },
+        scss: {
+          prependData: `@import '@mozilla-protocol/core/protocol/css/protocol.scss';`,
+        },
+        postcss: {
+          plugins: [autoPrefixer],
+        },
+      }),
+
+      // turn off warning about unused CSS selectors
```

Comment:
Can you give a little bit more context on why we're doing this, for the benefit of someone (maybe even you) looking at it in the future? 😄 

```suggestion
      // turn off warning about unused CSS selectors
```